kateryna/daemon/parsing_logs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+
+ advanced_syslog() {
+   function log {
+        while read LINE; do
+                  if [[ $LINE == "6"* ]]; then
```

Comment:
I changed  `/etc/rsyslog.conf` in order these levels to be shown like this `https://askubuntu.com/questions/79684/syslog-not-showing-log-levels-in-messages`